sdk/cosmos/azcosmos/cosmos_account_region_properties.go

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package azcosmos
+
+// The AccountRegion struct represents an Azure Cosmos DB database account in a specific region.
+type AccountRegion struct {
+	// Gets the name of the database account in the Azure Cosmos DB service.
+	Name string `json:"Constants.Properties.Name"`
```

Comment:
How does this json deserialization work? `json:....` is meant to map to the JSON payload attribute name.

Reply:
For example, if the Json is `{'name':'xxxxxx'}` then Go's Json comment would be `json: name`. See https://pkg.go.dev/encoding/json#Marshal